schedule/utils.py

```diff
@@ -64,7 +64,7 @@ class OccurrenceReplacer(object):
     """
 
     def __init__(self, persisted_occurrences):
-        lookup = [((occ.event.id, occ.original_start, occ.original_end), occ) for
+        lookup = [((occ.event.id, occ.start, occ.end), occ) for
```

Comment:
If a persisted occurrence has a modified start or end date, won't this cause the comparison to the generated occurrences to fail, which will end up having both the persisted and generated occurrences both appear on the calendar?

I believe the `original_start` and `original_end` avoids this problem.

Reply:
In our case, we need both appear in the calendar with their associated data(from a related model). So this just fits our need.

Reply:
I think you can close this PR. Thank for your fast response :)